samples/tutorial/Tutorial1.md

```diff
@@ -0,0 +1,239 @@
+# Step 1
+
+_Let's get something on the screen..._
+
+## Setup
+
+To follow this tutorial, launch Android Studio and open this folder (`samples/tutorial`).
+
+The `tutorial-base` module will be our starting place to build from.
+
+The welcome screen should look like:
+
+![Welcome](images/welcome.png)
+
+You can enter a name, but the login button won't do anything.
+
+## First Workflow
+
+Let's start by making a workflow and screen to back the welcome view.
+
+Start by creating a new workflow and screen by creating a new file with the [file templates](../../install-templates.sh), adding it to the `tutorial-base` module:
+
+![New Workflow](images/new-workflow.png)
+![Workflow Name](images/workflow-name.png)
+
+Follow the same steps using the `Layout Runner (ViewBinding)` template.
+
+### Screens, View Factories, and Layout Runners
+
+Let's start with what a "screen" is, and how it relates to the view controller.
+
+"Screen" is just the term we use to refer to a value type that represents the view model for a logical screen. Sometimes we'll even use the terms "screen" and "view model" interchangeably. It has no special type. Typically, a screen will be used as the rendering type for the workflow that is responsible for that screen. A screen is usually a data class, since that's the easiest way to make value type-like classes in Kotlin.
+
+For our welcome screen, we'll define what it needs for a backing view model:
+```kotlin
+data class WelcomeScreen(
+  /** The current name that has been entered. */
+  val name: String,
+  /** Callback when the name changes in the UI. */
+  val onNameChanged: (String) -> Unit,
+  /** Callback when the login button is tapped. */
+  val onLoginTapped: () -> Unit
+)
+```
+
+Then we need to create a `ViewFactory` that knows how to create an Android `View` to draw the actual screen. The easiest way to create a `ViewFactory` is to create a layout runner. A layout runner is a class that has a reference to the view and knows how to update the view given an instance of a screen. In a typical app, every screen will have a layout runner. Layout runners can also work with AndroidX `ViewBinding`s, which we'll use to define the `WelcomeLayoutRunner`. We have a pre-built `WelcomeViewBinding` that you can use, but if you would like to create and lay out the view yourself, feel free to do so!
```

Comment:
Shouldn't the first _layout runner_ instead be `LayoutRunner`?

Reply:
Isn't it? 
![image](https://user-images.githubusercontent.com/101754/99546403-3a393800-296b-11eb-9713-a3eaa735f9b1.png)

Reply:
I think he means the class reference, not the general noun.

Reply:
@rjrjr Can you elaborate? I'm very confused.